ci/testnet-manager.sh

```diff
@@ -241,19 +241,19 @@ start() {
         EC2_ZONE_ARGS+=("-z $val")
       done
 
-      [[ -n $EC2_NODE_COUNT ]] || EC2_NODE_COUNT=60
-      [[ -n $GCE_NODE_COUNT ]] || GCE_NODE_COUNT=40
-
       # shellcheck disable=SC2068
-      ci/testnet-deploy.sh -p beta-testnet-solana-com -C ec2 ${EC2_ZONE_ARGS[@]} \
-        -t "$CHANNEL_OR_TAG" -n "$EC2_NODE_COUNT" -c 0 -s -u -P -a eipalloc-0f286cf8a0771ce35 \
+      [[ -z $EC2_NODE_COUNT ]] || ci/testnet-deploy.sh -p beta-testnet-solana-com -C ec2 ${EC2_ZONE_ARGS[@]} \
+        -t "$CHANNEL_OR_TAG" -n "$EC2_NODE_COUNT" -c 0 -u -P -a eipalloc-0f286cf8a0771ce35 \
         ${maybeReuseLedger:+-r} \
-        ${maybeDelete:+-D}
+        ${maybeDelete:+-D} \
+        ${GCE_NODE_COUNT:+-s}
```

Comment:
Should this be EC2_NODE_COUNT?  Looks like the providers are reversed here and at ln#256

Reply:
No, the purpose is to check if GCE nodes are also configured. "-s" specifies whether to wait for more nodes, or start the network.

Reply:
GOt it thanks